indexd/blueprint.py

```diff
@@ -65,50 +57,11 @@ def get_record(record):
             try:
                 ret = blueprint.alias_driver.get(record)
             except AliasNoRecordFound:
-                if not blueprint.dist or "no_dist" in flask.request.args:
-                    raise
-                ret = dist_get_record(record)
+                raise IndexNoRecordFound("no record found")
 
     return flask.jsonify(ret), 200
 
 
-def dist_get_record(record):
-
-    # Sort the list of distributed ID services
-    # Ones with which the request matches a hint will be first
-    # Followed by those that don't match the hint
-    sorted_dist = sorted(
-        blueprint.dist, key=lambda k: hint_match(record, k["hints"]), reverse=True
-    )
-
-    for indexd in sorted_dist:
-        try:
-            if indexd["type"] == "doi":
-                fetcher_client = DOIClient(baseurl=indexd["host"])
-                res = fetcher_client.get(record)
-            elif indexd["type"] == "dos":
-                fetcher_client = DOSClient(baseurl=indexd["host"])
-                res = fetcher_client.get(record)
-            else:
-                res = requests.get(
```

Comment:
I just want to confirm. The bio team uses a one of the functions of indexd to query against the bioindexd vm, and if the document doesn't exist there bioindexd queries the indexd vm (gdc indexd in the same cluster). This isn't removing that functionality right?

Reply:
Thank you!

recovered the function.